encoder/src/main/java/openlr/encoder/postprocessor/OfframpProcessor.java

```diff
@@ -0,0 +1,97 @@
+package openlr.encoder.postprocessor;
+
+import openlr.OpenLRProcessingException;
+import openlr.encoder.properties.OpenLREncoderProperties;
+import openlr.map.GeoCoordinates;
+import openlr.map.Line;
+import openlr.map.Node;
+import openlr.map.utils.GeometryUtils;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class OfframpProcessor extends AbstractLrpProcessor {
+
+    private static double BEARING_RANGE_OF_SIBLING_LINES = 45;
+
+    public static class RouteNodeInfo {
+        public Node node;
+        public double lengthViaNode;
+        public RouteNodeInfo(Node node, double lengthViaSNode) {
```

Comment:
is the `S` in the `lengthViaSNode` a typo?

Reply:
f6c41f62659ef9375da2b7438f0985e9587cb108